CourseApp.Tests/CalculatingAPersonsAgeTests.cs

```diff
@@ -0,0 +1,24 @@
+namespace CourseApp.Tests
+{
+    using System;
+    using Xunit;
+
+    public class CalculatingAPersonsAgeTests
+    {
+        [Theory]
+        [InlineData(new[] { 2002, 12, 20 }, new[] { 2021, 12, 20 }, "Congratulations on your 19th birthday !!!")]
```

Comment:
🔕 
you may use named tuples here - https://docs.microsoft.com/ru-ru/dotnet/csharp/language-reference/builtin-types/value-tuples#tuple-field-names